diff --git a/dist/index.js b/dist/index.js
index b7355c6c53fb93f21c53e8a6c521fb4b029e12c5..0c266b3bde060103f02d54d8cc350f0b8bac48ef 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -38,7 +38,7 @@ const runBlockchain_1 = __importDefault(require("./runBlockchain"));
 const AsyncEventEmitter = require('async-eventemitter');
 const promisify = require('util.promisify');
 // very ugly way to detect if we are running in a browser
-const isBrowser = new Function('try {return this===window;}catch(e){ return false;}');
+const isBrowser = new Function('return true;');
 let mcl;
 let mclInitPromise;
 if (!isBrowser()) {
diff --git a/dist/state/stateManager.js b/dist/state/stateManager.js
index 81091ae552fda6e8ce24ef27e8d3d4ff63842918..13bc64fe0ca1d5011e44eed40377521d46488488 100644
--- a/dist/state/stateManager.js
+++ b/dist/state/stateManager.js
@@ -22,7 +22,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-const Set = require('core-js-pure/es/set');
+//const Set = require('core-js-pure/es/set');
 const debug_1 = require("debug");
 const merkle_patricia_tree_1 = require("merkle-patricia-tree");
 const ethereumjs_util_1 = require("ethereumjs-util");
diff --git a/dist.browser/index.js b/dist.browser/index.js
index e4955883b1798d54b1cdedf415d1b25a27c9d366..8f029c83db26176f2aead8fe216d628874b050cb 100644
--- a/dist.browser/index.js
+++ b/dist.browser/index.js
@@ -100,7 +100,7 @@ var runBlockchain_1 = __importDefault(require("./runBlockchain"));
 var AsyncEventEmitter = require('async-eventemitter');
 var promisify = require('util.promisify');
 // very ugly way to detect if we are running in a browser
-var isBrowser = new Function('try {return this===window;}catch(e){ return false;}');
+var isBrowser = new Function('return true;');
 var mcl;
 var mclInitPromise;
 if (!isBrowser()) {
diff --git a/dist.browser/state/stateManager.js b/dist.browser/state/stateManager.js
index 838ca7866444b2ee54a09d280ed390bd63681cd4..3b744dabf62560e28f0815f6ffef70ce8b089ab2 100644
--- a/dist.browser/state/stateManager.js
+++ b/dist.browser/state/stateManager.js
@@ -94,7 +94,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-var Set = require('core-js-pure/es/set');
+//var Set = require('core-js-pure/es/set');
 var debug_1 = require("debug");
 var merkle_patricia_tree_1 = require("merkle-patricia-tree");
 var ethereumjs_util_1 = require("ethereumjs-util");
