diff --git a/_root.js b/_root.js
index d2852bed4d2e469b3b87ed31df2350047e005ff4..18b86821f36625c4a585477cea239e153c477db6 100644
--- a/_root.js
+++ b/_root.js
@@ -1,9 +1,3 @@
-var freeGlobal = require('./_freeGlobal');
-
-/** Detect free variable `self`. */
-var freeSelf = typeof self == 'object' && self && self.Object === Object && self;
-
-/** Used as a reference to the global object. */
-var root = freeGlobal || freeSelf || Function('return this')();
+var root = globalThis;
 
 module.exports = root;
